ush/parsing_namelists_CICE.sh

```diff
@@ -4,371 +4,594 @@ CICE_namelists(){
+   f_tmask         = .true.
+   f_blkmask       = .true.
+   f_tarea         = .true.
+   f_uarea         = .true.
+   f_dxt           = .false.
+   f_dyt           = .false.
+   f_dxu           = .false.
+   f_dyu           = .false.
+   f_HTN           = .false.
+   f_HTE           = .false.
+   f_ANGLE         = .true.
+   f_ANGLET        = .true.
+   f_NCAT          = .true.
+   f_VGRDi         = .false.
+   f_VGRDs         = .false.
+   f_VGRDb         = .false.
+   f_VGRDa         = .true.
+   f_bounds        = .false.
+   f_aice          = 'mdhxx' 
+   f_hi            = 'mdhxx'
+   f_hs            = 'mdhxx' 
+   f_Tsfc          = 'mdhxx' 
+   f_sice          = 'mdhxx' 
+   f_uvel          = 'mdhxx' 
+   f_vvel          = 'mdhxx'
+   f_uatm          = 'mdhxx'
+   f_vatm          = 'mdhxx'
+   f_fswdn         = 'mdhxx' 
+   f_flwdn         = 'mdhxx'
+   f_snowfrac      = 'x'
+   f_snow          = 'mdhxx' 
+   f_snow_ai       = 'x' 
+   f_rain          = 'mdhxx' 
+   f_rain_ai       = 'x' 
+   f_sst           = 'mdhxx' 
+   f_sss           = 'mdhxx' 
+   f_uocn          = 'mdhxx' 
+   f_vocn          = 'mdhxx' 
+   f_frzmlt        = 'mdhxx'
+   f_fswfac        = 'mdhxx'
+   f_fswint_ai     = 'x'
+   f_fswabs        = 'mdhxx' 
+   f_fswabs_ai     = 'x' 
+   f_albsni        = 'mdhxx' 
+   f_alvdr         = 'mdhxx'
+   f_alidr         = 'mdhxx'
+   f_alvdf         = 'mdhxx'
+   f_alidf         = 'mdhxx'
+   f_alvdr_ai      = 'x'
+   f_alidr_ai      = 'x'
+   f_alvdf_ai      = 'x'
+   f_alidf_ai      = 'x'
+   f_albice        = 'mdhxx'
+   f_albsno        = 'mdhxx'
+   f_albpnd        = 'mdhxx'
+   f_coszen        = 'mdhxx'
+   f_flat          = 'mdhxx' 
+   f_flat_ai       = 'x' 
+   f_fsens         = 'mdhxx' 
+   f_fsens_ai      = 'x' 
+   f_fswup         = 'x'
+   f_flwup         = 'mdhxx' 
+   f_flwup_ai      = 'x' 
+   f_evap          = 'mdhxx' 
+   f_evap_ai       = 'x' 
+   f_Tair          = 'mdhxx' 
+   f_Tref          = 'mdhxx' 
+   f_Qref          = 'mdhxx'
+   f_congel        = 'mdhxx' 
+   f_frazil        = 'mdhxx' 
+   f_snoice        = 'mdhxx' 
+   f_dsnow         = 'mdhxx' 
+   f_melts         = 'mdhxx'
+   f_meltt         = 'mdhxx'
+   f_meltb         = 'mdhxx'
+   f_meltl         = 'mdhxx'
+   f_fresh         = 'mdhxx'
+   f_fresh_ai      = 'x'
+   f_fsalt         = 'mdhxx'
+   f_fsalt_ai      = 'x'
+   f_fbot          = 'mdhxx'
+   f_fhocn         = 'mdhxx' 
+   f_fhocn_ai      = 'x' 
+   f_fswthru       = 'mdhxx' 
+   f_fswthru_ai    = 'x' 
+   f_fsurf_ai      = 'x'
+   f_fcondtop_ai   = 'x'
+   f_fmeltt_ai     = 'x' 
+   f_strairx       = 'mdhxx' 
+   f_strairy       = 'mdhxx' 
+   f_strtltx       = 'x' 
+   f_strtlty       = 'x' 
+   f_strcorx       = 'x' 
+   f_strcory       = 'x' 
+   f_strocnx       = 'mdhxx' 
+   f_strocny       = 'mdhxx' 
+   f_strintx       = 'x' 
+   f_strinty       = 'x'
+   f_taubx         = 'x'
+   f_tauby         = 'x'
+   f_strength      = 'x'
+   f_divu          = 'mdhxx'
```

Comment:
I see that the settings for f_divu, f_shear, f_dvidtt, f_dvidtd, f_daitt, f_daidtd have been copied from the previous settings. I really can't imagine these being useful in the current context and I would remove them (change 'mdhxx' to 'x'.

Reply:
Sorry I missed that.